Packs/Tenable_io/Integrations/Tenable_io/Tenable_io.py

```diff
@@ -496,18 +496,16 @@ def get_scan_status_command():
 
 
 def is_valid_scan_id(scan_id):
-    if scan_id == "" or scan_id.isalpha():
-        return False
-    return True
+    return not (scan_id == "" or scan_id.isalpha())
```

Comment:
@barchen1 If you want to verify this is a number, `isalpha()` won't work, you need `isdigit()`
![image](https://user-images.githubusercontent.com/33782301/111335898-6cd71100-867d-11eb-891d-bf9fbc691cef.png)